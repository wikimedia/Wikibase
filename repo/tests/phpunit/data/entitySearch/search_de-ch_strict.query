{
	"search": "Wien",
	"language": "de-ch",
	"type": "item",
	"strictlanguage": true
}
