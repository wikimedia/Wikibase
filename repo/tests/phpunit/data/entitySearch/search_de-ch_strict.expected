{
    "query": {
        "bool": {
            "should": [
                {
                    "bool": {
                        "filter": [
                            {
                                "match": {
                                    "labels_all.prefix": "Wien"
                                }
                            }
                        ],
                        "must": [
                            {
                                "multi_match": {
                                    "type": "best_fields",
                                    "tie_breaker": 0,
                                    "query": "Wien",
                                    "fields": [
                                        "labels.de-ch.near_match^40",
                                        "labels.de-ch.near_match_folded^30",
                                        "labels.de-ch.prefix^15"
                                    ]
                                }
                            }
                        ]
                    }
                },
                {
                    "term": {
                        "title.keyword": "Wien"
                    }
                }
            ],
            "minimum_number_should_match": 1,
            "filter": [
                {
                    "term": {
                        "content_model": "wikibase-item"
                    }
                }
            ]
        }
    },
    "_source": [
        "namespace",
        "title",
        "labels.de-ch",
        "descriptions.de-ch",
        "labels.de",
        "descriptions.de",
        "labels.en",
        "descriptions.en"
    ],
    "fields": [],
    "highlight": {
        "pre_tags": [
            ""
        ],
        "post_tags": [
            ""
        ],
        "fields": {
            "title": {
                "type": "experimental",
                "fragmenter": "none",
                "number_of_fragments": 0,
                "matched_fields": [
                    "title.keyword"
                ]
            },
            "labels.de-ch.prefix": {
                "type": "experimental",
                "fragmenter": "none",
                "number_of_fragments": 0,
                "options": {
                    "skip_if_last_matched": true
                }
            },
            "labels.de.prefix": {
                "type": "experimental",
                "fragmenter": "none",
                "number_of_fragments": 0,
                "options": {
                    "skip_if_last_matched": true
                }
            },
            "labels.en.prefix": {
                "type": "experimental",
                "fragmenter": "none",
                "number_of_fragments": 0,
                "options": {
                    "skip_if_last_matched": true
                }
            },
            "labels_all.prefix": {
                "type": "experimental",
                "fragmenter": "none",
                "number_of_fragments": 0,
                "options": {
                    "skip_if_last_matched": true
                }
            }
        }
    },
    "rescore": [
        {
            "window_size": 8192,
            "query": {
                "query_weight": 1,
                "rescore_query_weight": 1,
                "score_mode": "multiply",
                "rescore_query": {
                    "function_score": {
                        "score_mode": "max",
                        "functions": [
                            {
                                "field_value_factor": {
                                    "field": "label_count",
                                    "missing": 0
                                },
                                "weight": 1
                            },
                            {
                                "field_value_factor": {
                                    "field": "sitelink_count",
                                    "missing": 0
                                },
                                "weight": 1
                            }
                        ]
                    }
                }
            }
        }
    ]
}